content/v2/openapi.yml

```diff
@@ -2963,6 +3083,21 @@ components:
           - 'label:desc'
           - 'email:asc'
           - 'email:desc'
+    SortDNSAnalytics:
+      description: Sort results. Default sorting policy is by ascending date, then ascending zone_name.
+      name: sort
+      style: form
+      in: query
+      required: false
+      schema:
+        type: string
+        enum:
+          - 'date:asc'
+          - 'date:desc'
+          - 'volume:asc'
+          - 'volume:desc'
+          - 'zone_name:asc'
+          - 'zone_name:desc'
```

Comment:
Multiple sorting criteria can be defined. In fact, the default is `"date:asc,zone_name:asc"`. Can we just not provide the `enum` here?

Reply:
I just followed the example of other endpoints in which we are providing the enum, even when the values can be combined. Not sure if I did the right thing.

Reply:
I'm not sure how to interpret `enum` either...

Reply:
I _think_ that we're not using them correctly if values can be combined https://swagger.io/docs/specification/data-models/enums/

It sounds like we would need to list all permutations under all possible sorting, which feels wrong to do. Maybe we should not have `enum` unless there's a finite list of values our customers need to pick from.

Reply:
I was looking [at the same thing](https://swagger.io/docs/specification/data-models/enums/) and I agree: I do not think this is the correct use. Will update this momentarily.

Reply:
See: [1de58d1](https://github.com/dnsimple/dnsimple-developer/pull/558/commits/1de58d1563f6ca6a2cbc214d0bdf9aaa63f7b038)